Packages/vcs/Lib/configurator.py

```diff
@@ -730,155 +664,42 @@ def create(self):
             x.append(point[0] / float(w))
             y.append(point[1] / float(h))
 
-        created = None
-
-        if self.creating == CREATING_FILL:
-            fill = self.canvas.createfillarea()
-            fill.x = x
-            fill.y = y
-            created = fill
-            self.fill_button.set_state(0)
-            dp = self.canvas.fillarea(fill)
-        elif self.creating == CREATING_TEXT:
+        if self.creating == CREATING_TEXT:
             t = self.canvas.createtextcombined()
             t.x = x
             t.y = y
-            t.string = ["New Text"]
-            created = t
+            t.string = ["Click to Edit"]
             self.text_button.set_state(0)
             dp = self.canvas.text(t)
+            key = "vtk_backend_text_actors"
         elif self.creating == CREATING_MARKER:
             m = self.canvas.createmarker()
             m.x = x
             m.y = y
             m.size = [10]
-            created = m
             self.marker_button.set_state(0)
             dp = self.canvas.marker(m)
-        elif self.creating == CREATING_LINE:
-            l = self.canvas.createline()
-            l.x = x
-            l.y = y
-            created = l
-            self.line_button.set_state(0)
-            dp = self.canvas.line(l)
+            key = "vtk_backend_marker_actors"
 
         # Activate an editor for the new object
-        self.clicked_info = 0
-        self.activate(created, dp)
+        self.activate(dp, dp.backend[key][0], key)
 
         self.creating = False
         self.click_locations = None
 
+def get_attribute(display, backend_key):
+    key = backend_key.split("_")[2]
+    template = t(display.template)
+    if key in ("Min", "Max", "Mean"):
+        return getattr(template, key.lower())
+    else:
+        return getattr(template, key)
 
 def t(name):
```

Comment:
This is old code, but single character identifiers are a bad idea: It's hard to tell what the function/variable does when it's used, and it will be easily shadowed by temporary/locally scoped variables. Let's avoid these in the future.